config/hardware/addons/accelerometers/adxl345_sht.cfg

```diff
@@ -0,0 +1,15 @@
+[adxl345]
+cs_pin: SHTCan:ADXL_CS
```

Comment:
In the same way, can you change SHTCan by "toolhead". I think, this will allow more genericity.

Reply:
Done